specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/definitions.json

```diff
@@ -4567,6 +4567,114 @@
       ],
       "description": "Policy contract Properties."
     },
+    "PolicyWithComplianceCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyWithComplianceContract"
+          },
+          "description": "Policy Contract value."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "The response of the list policy operation."
+    },
+    "PolicyWithComplianceContract": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyWithComplianceContractProperties",
+          "description": "Properties of the Policy."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Policy Contract details."
+    },
+    "PolicyWithComplianceContractProperties": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Contents of the Policy as defined by the format."
+        },
+        "format": {
+          "type": "string",
+          "description": "Format of the policyContent.",
+          "enum": [
+            "xml",
+            "xml-link",
+            "rawxml",
+            "rawxml-link"
+          ],
+          "x-ms-enum": {
+            "name": "PolicyContentFormat",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "xml",
+                "description": "The contents are inline and Content type is an XML document."
+              },
+              {
+                "value": "xml-link",
+                "description": "The policy XML document is hosted on a HTTP endpoint accessible from the API Management service."
+              },
+              {
+                "value": "rawxml",
+                "description": "The contents are inline and Content type is a non XML encoded policy document."
+              },
+              {
+                "value": "rawxml-link",
+                "description": "The policy document is not XML encoded and is hosted on a HTTP endpoint accessible from the API Management service."
+              }
+            ]
+          },
+          "default": "xml"
+        },
+        "policyId": {
+          "type": "string",
+          "description": "Policy Identifier"
+        },
```

Comment:
Given policyId is part of resource "id" and is in the resource Url, it is not needed here. Please remove it.